### ISSUE TITLE
Static create method

### ISSUE THREAD

**Issue by djgadd** [PhpUnitsOfMeasure/php-units-of-measure#47]
Thanks for the solid library.

What do you think about a static create method in the AbstractPhysicalQuantity class? Allowing: -

``` php
Length::create(6.16, 'feet')->toUnit('m');
```

It's obviously nothing major, I've just got a few cases where I begrudge having an extra line just to declare the Length object when all I want is a simple conversion.

**Comment by ste93cry** [comment]
:+1:

**Comment by fyrye** [comment]
In php 5.4+ you really don't need an extra line, you can wrap instantiators in parenthesis to chain method calls.
See: http://php.net/manual/en/migration54.new-features.php
```php
echo (new Length(6.16, 'ft'))->toUnit('m');
```

**Comment by ste93cry** [comment]
This is true, but a static method feels more appropriate and the legibility of the code is better imho

**Comment by triplepoint** [comment]
Sorry for not addressing this ticket earlier.

It's a style judgement, and therefore somewhat arbitrary, but I'd prefer to lean toward existing language semantics rather than code around them.  @fyrye's suggestion above is what I'd consider to be the recommended procedure for a one-liner unit conversion.

**Comment by ste93cry** [comment]
I would like to give an example of when the static method could be useful: in some cases you have to instantiate a lot of `Length` objects in a loop, maybe using `array_map` (think maybe to create instances out of a SQL result set). This is not possible without a static method or a lambda function that invokes the constructor, and it would be prettier to write something like `array_map([Length::class, 'create'], ...)`

**Comment by triplepoint** [comment]
Hmm, ok it looks like there are legit use cases here, and the added complexity is minimal. I'm ok with implementing this change.
